### ISSUE TITLE
Post API call needs more data returned

### ISSUE THREAD

**Issue by NathanYocum** [LectureGoggles/FlaskAPI#29]
For the post we need:
```
 

- [✓]  subject, // name and probably id as well
- [✓]   topic, // name
- [✓]   author, // name
- [✓]   authorImg, // (zack) complete
- [✓]   previewImg, // (zack) currently working on
- [✓]   points, // (zack) currently working on

```

**Comment by zackjohnson8** [comment]
For the names on subject, topic, and author, did you want the names of the users who created these?

**Comment by NathanYocum** [comment]
Probably not, it'll just need the text to display on the frontend. The subject id would be nice to have for filtering and stuff like that

**Comment by zackjohnson8** [comment]
So when the object is created return the information. Will work on that now then.

**Comment by NathanYocum** [comment]
Should have made this more clear, I need this on the `/v1/post/getAll` and other `/v1/post` calls.

Upvote count is there now, but I need the author, subject, and topic (only the names) from those calls. This is to make [resourceCard.jsx](https://github.com/LectureGoggles/lecture-goggles-frontend/blob/master/src/components/resourceCard/ResourceCard.jsx) have the info it needs without multiple API calls.

**Comment by zackjohnson8** [comment]
Gotcha, thanks for the clarification. Working on fixing now.

**Comment by NathanYocum** [comment]
Closed by #42
